### PR TITLE
Smartsheet Connector -New branch for clean PR

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: smartsheet/1.0/antora.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: smartsheet-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: Smartsheet Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to programmatically access and manage your organization''s Smartsheet data and account information
```
Comment (kevintroller) [inline smartsheet/1.0/antora.yml]:
```suggestion
    page-component-desc: Enables you to programmatically access and manage your organization's Smartsheet data and account information
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,1378 @@
...
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the connection Idle Timeout |  SECONDS |
+| Proxy Config a| <<Proxy>> |  Reusable configuration element for outbound connections through a proxy |  |
+| Stream Response a| Boolean |
+Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed, and the body is streamed as it is received.
+ |  false |
+| Response Buffer Size a| Number |  The space, in bytes, for the buffer where the HTTP response will be stored. |  -1 |
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc:75]:
```suggestion
| Response Buffer Size a| Number |  The space, in bytes, for the buffer to store the HTTP response. |  -1 |
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,1378 @@
...
+| Connection Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Connection Timeout |  SECONDS |
+| Use Persistent Connections a| Boolean |  If false, each connection will be closed after the first request is completed. |  true |
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc:59]:
```suggestion
| Use Persistent Connections a| Boolean |  If false, each connection is closed after the first request is completed. |  true |
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,1378 @@
...
+
+
+== Operations
+
+[[CreateSheets]]
+== Create Sheet
+`<smartsheet:create-sheets>`
+
+
+Creates a new sheet in the user's Sheets folder. This operation makes an HTTP POST request to the /sheets endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc:124]:
```suggestion
Creates a new sheet in the user's Sheets folder. This operation makes an HTTP POST request to the sheets endpoint.
```

Comment (kevintroller) [reply smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc:124]: or `/sheets`

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSheetsCopyBySheetId]]
+== Copy Sheet
+`<smartsheet:create-sheets-copy-by-sheet-id>`
+
+
+Creates a copy of the specified sheet. This operation makes an HTTP POST request to the /sheets/{sheetId}/copy endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Creates a copy of the specified sheet. This operation makes an HTTP POST request to the `/sheets/{sheetId}/copy` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSheetsDiscussionsCommentsBySheetIdDiscussionId]]
+== Add Comment
+`<smartsheet:create-sheets-discussions-comments-by-sheet-id-discussion-id>`
+
+
+Adds a comment to a discussion. This operation makes an HTTP POST request to the /sheets/{sheetId}/discussions/{discussionId}/comments endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Adds a comment to a discussion. This operation makes an HTTP POST request to the `/sheets/{sheetId}/discussions/{discussionId}/comments` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 7]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSheetsRowsAttachmentsBySheetIdRowId]]
+== Add File to Row
+`<smartsheet:create-sheets-rows-attachments-by-sheet-id-row-id>`
+
+
+Attaches a file to the specified row. This operation makes an HTTP POST request to the /sheets/{sheetId}/rows/{rowId}/attachments endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Attaches a file to the specified row. This operation makes an HTTP POST request to the `/sheets/{sheetId}/rows/{rowId}/attachments` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 8]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSheetsRowsBySheetId]]
+== Add Rows
+`<smartsheet:create-sheets-rows-by-sheet-id>`
+
+
+Inserts one or more rows into the specified sheet. This operation makes an HTTP POST request to the /sheets/{sheetId}/rows endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Inserts one or more rows into the specified sheet. This operation makes an HTTP POST request to the `/sheets/{sheetId}/rows` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 9]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSheetsRowsCopyBySheetId]]
+== Copy Rows to Another Sheet
+`<smartsheet:create-sheets-rows-copy-by-sheet-id>`
+
+
+Copies rows from the specified sheet to the bottom of another sheet. This operation makes an HTTP POST request to the /sheets/{sheetId}/rows/copy endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Copies rows from the specified sheet to the bottom of another sheet. This operation makes an HTTP POST request to the `/sheets/{sheetId}/rows/copy` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 10]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[DeleteSheetsBySheetId]]
+== Delete Sheet
+`<smartsheet:delete-sheets-by-sheet-id>`
+
+
+Deletes the specified sheet. This operation makes an HTTP DELETE request to the /sheets/{sheetId} endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Deletes the specified sheet. This operation makes an HTTP DELETE request to the `/sheets/{sheetId}` endpoint.
```

File: smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc [hunk 11]
```diff
@@ -0,0 +1,1378 @@
...
+* SMARTSHEET:UNAUTHORIZED
+* SMARTSHEET:UNSUPPORTED_MEDIA_TYPE
+
+
+[[DeleteSheetsRowsBySheetId]]
+== Delete Rows
+`<smartsheet:delete-sheets-rows-by-sheet-id>`
+
+
+Deletes one or more rows from the specified sheet. This operation makes an HTTP DELETE request to the /sheets/{sheetId}/rows endpoint.
```
Comment (kevintroller) [inline smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]:
```suggestion
Deletes one or more rows from the specified sheet. This operation makes an HTTP DELETE request to the `/sheets/{sheetId}/rows` endpoint.
```

Comment (kevintroller) [reply smartsheet/1.0/modules/ROOT/pages/smartsheet-connector-reference.adoc]: DITTO GLOBAL

**Review by kevintroller — DISMISSED** [review]
LGTM, minor comments

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**